sqlmesh/core/engine_adapter/snowflake.py

```diff
@@ -59,7 +59,8 @@ def query_factory() -> Query:
                 self._connection_pool.get(),
                 df,
                 temp_table.name,
-                schema=temp_table.db,
+                schema=temp_table.db or None,
```

Comment:
this doesn't respect quotes, should it be temp_table.args["db"].sql(dialect=snowflake) ?